modules/Campaigns/CampaignDiagnostic.php

```diff
@@ -59,21 +55,15 @@
     $params = array();
     $params[] = "<a href='index.php?module=Campaigns&action=index'>{$mod_strings['LBL_MODULE_NAME']}</a>";
     $params[] = $mod_strings['LBL_CAMPAIGN_DIAGNOSTICS'];
-    
+
     echo getClassicModuleTitle('Campaigns', $params, true);
 }
 
 global $theme;
 global $currentModule;
 
 
-
-
-if (isset($_REQUEST['inline']) && $_REQUEST['inline'] == 'inline') {
-    {
-
-}
-} else {
+if (!isset($_REQUEST['inline']) && $_REQUEST['inline'] !== 'inline') {
```

Comment:
Hi @Dillon-Brown,

I think here the expressions should use `||` instead of `&&`: `!isset($_REQUEST['inline']) || $_REQUEST['inline'] !== 'inline'`. What do you think?

Reply:
Agreed, pushed!